src/scss/config/_scale.scss

```diff
@@ -87,12 +81,7 @@ $theme-sizes: (
 );
 
 @include tools.add-sizes($theme-sizes);
-@include herman.add(
-  'sizes',
-  'theme-sizes',
-  $theme-sizes,
-  get-function('size', $module: 'tools')
-);
+@include herman.add('sizes', 'theme-sizes', tools.compile-sizes($theme-sizes));
```

Comment:
Something seems to be causing an unwanted change to how the `border-width` size is displayed:

Before (https://www.oddbird.net/styleguide/config-scale#variable--theme-sizes):

![Screen Shot 2022-01-24 at 4 40 16 PM](https://user-images.githubusercontent.com/552316/150869691-e17cdba2-dae3-49e8-929f-d5c2a79fcf64.jpg)

After (https://deploy-preview-187--oddleventy.netlify.app/styleguide/config-scale#variable--theme-sizes):

![Screen Shot 2022-01-24 at 4 40 12 PM](https://user-images.githubusercontent.com/552316/150869708-6bd126cb-6fee-48b3-ab36-090d20c59043.jpg)

Reply:
Interesting. Both are wrong…

Herman doesn't currently support visualizing named lengths like this. I filed a card to fix it in Herman: https://trello.com/c/gAFt8dMk/59-herman-how-should-length-keywords-like-a-medium-border-width-be-previewed

Reply:
@jgerigmeyer Hmm, before this I think it was using the font-size reference value for `medium` instead of the CSS keyword. 

@mirisuzanne until this is supported in Herman, should we replace the keyword value with just a raw length value (3px)?

Reply:
I don't feel like we need to adjust our styles to match what Herman supports, but I also don't think it hurts much if you want. ¯\\\_(ツ)_/¯